my Journey - Shourgamer2! (#43)
collab-community/journey-book: journeys/shourgamer2.md

### Shourgamer2's Journey

> I started coding in 2020. It was lockdown and it was boring to play games all day so I decided to learn to code. My first language I bealive was python or C#. 

Reviewer: ```suggestion
## How I Began Coding!
I started coding in 2020. It was lockdown and it was boring to play games all day so I decided to learn to code. My first language I bealive was python or C#. 
```

> I was intrested in making games so I learnt the unity engine and started using it. I used to code all day at that time and I was working on a project called the "Golden pistol"

Reviewer: ```suggestion
I was intrested in making games so I learnt the [unity engine](https://unity.com) and started using it. I used to code all day at that time and I was working on a project called the "Golden pistol"
```

link the unity engine website for people that haven't heard of it.

Which was a fps story game. Sadly after I exported the game I didn't publish it. Well then I got intrest in roblox games (which I have tried before but at that time I thought it will be a waste of time to make games in roblox)
So I built roblox games and I watched some youtube videos to learn lua.
### What I have done so far! 

> I have built a lot of projects like [TermBolt - A terminal ](https://github.com/shourgamer2/termbolt), [What-Language -A programming language (In work, uses the terminal for most of the operation. Like a toy language for now)](https://github.com/what-language)

Reviewer: ```suggestion
I have built a lot of projects like [TermBolt - A terminal ](https://github.com/shourgamer2/termbolt), [What-Language -A programming language (In work, uses the terminal for most of the operation. Like a toy language for now)](https://github.com/what-language).
```

fix grammer issue

## How I Began Coding!

> I started coding in 2020. It was lockdown and it was boring to play games all day so I decided to learn to code. My first language I bealive was python or C#. 

Reviewer: ```suggestion
I started coding in 2020. It was lockdown and it was boring to play games all day so I decided to learn to code. The first language that I learnt to code was either Python or C#, I'm unsure as of this point.
```

 fix grammer

### Shourgamer2's Journey

> I was intrested in making games so I learnt the [unity engine](https://unity.com/) and started using it. I used to code all day at that time and I was working on a project called the "Golden pistol"

Reviewer: ```suggestion
I was intrested in making games so I learnt the [unity engine](https://unity.com) and started using it. I used to code all day at that time and I was working on a project called the "Golden pistol"
```

remove extra slash

> Which was a fps story game. Sadly after I exported the game I didn't publish it. Well then I got intrest in roblox games (which I have tried before but at that time I thought it will be a waste of time to make games in roblox)

Reviewer: ```
Well then I got intrest in roblox games (which I have tried before but at that time I thought it will be a waste of time to make games in roblox)
```

can you link your roblox dev page?

Reviewer: If you don't have a roblox dev page then no worries, If you link the page, then people can see what games you've made!

> Which was a fps story game. Sadly after I exported the game I didn't publish it. Well then I got intrest in roblox games (which I have tried before but at that time I thought it will be a waste of time to make games in roblox)

Reviewer: ```suggestion
Which was a fps story game. Sadly after I exported the game I didn't publish it. Well then I got interest in Roblox games (which I have tried before but at that time I thought it will be a waste of time to make games in Roblox)
```

fix grammer issues

> So I built roblox games and I watched some youtube videos to learn lua.

Reviewer: ```suggestion
So I built Roblox games and I watched some YouTube videos to learn Lua.
```

fix grammer issues

### What I have done so far! 
I have built a lot of projects like [TermBolt - A terminal ](https://github.com/shourgamer2/termbolt), [What-Language -A programming language (In work, uses the terminal for most of the operation. Like a toy language for now)](https://github.com/what-language).

> And more on https://shourgamer2.tk/#projects and I also own the team [ShourGames](https://github.com/shourgames)

Reviewer: ```suggestion
And more on https://shourgamer2.tk/#projects and I also own the team [ShourGames](https://github.com/shourgames).
```

### How to reach me?

> Fell free to email me at contact@shourgamer2.tk and you can also contact me at https://shourgamer2.tk/#contact

Reviewer: ```suggestion
Feel free to email me at contact@shourgamer2.tk and you can also contact me at https://shourgamer2.tk/#contact
```

fix grammer issue
